src/client/pages/Module.tsx

```diff
@@ -66,6 +58,9 @@ export default function Module() {
             />
           </>
         )}
+        <Button style={{ backgroundColor: getColor(View.COURSES) }}>
```

Comment:
Buttons can be customized to use react-router-dom links - let me post a snippet on how to use them in the main comments section of this PR.